pinchy/src/tests/mod.rs

```diff
@@ -3,9 +3,11 @@
 
 mod basic_io;
 mod filesystem;
+mod formatting_integration;
```

Comment:
This is not really an integration test, it's not real output being read from an execution of pinchy. We have the existing syscall parsing tests to verify return values being properly formatted, you'll need to update those expectations, any way. This file is not necessary.